x-pack/plugins/reporting/server/export_types/printable_pdf/lib/get_custom_logo.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ReportingCore } from '../../../';
+import { UI_SETTINGS_CUSTOM_PDF_LOGO } from '../../../../common/constants';
+import { ConditionalHeaders } from '../../../types';
+
+export const getCustomLogo = async (
```

Comment:
Had to clean up this file (besides moving it out of `common`) - it had a few unused params, and needed the spaceId param added.